varnish-macros/src/generator.rs

```diff
@@ -115,7 +115,12 @@ impl Generator {
     }
 
     fn gen_json(&self) -> String {
-        let mut header: Vec<Value> = vec!["$VMOD".into(), "1.0".into()];
+        let mut version = "2.0";
```

Comment:
same thing here - we don't want to introduce a mutable variable just because we have conditional compilation.  A better pattern might simply be something like this (if it works):

```rust
let mut header: Vec<Value> = vec![
    "$VMOD".into(),
    #[cfg(varnishsys_vmod_meta_1_0)]
    "1.0".into(),
    #[cfg(not(varnishsys_vmod_meta_1_0))]
    "2.0".into(),
];
```

Reply:
Yes - that works. Updated!

Reply:
just for completness sake, this pattern also works:

```rust
let version = if cfg!(varnishsys_vmod_meta_1_0) { "1.0" } else { "2.0" };
```